test/user.test.js

```diff
@@ -363,6 +363,88 @@ describe('User', function() {
     });
   });
 
+  describe('Password length validation', function() {
+    var pass72Char = new Array(73).join('a');
+    var pass73Char = new Array(74).join('a');
+    var passTooLong = pass72Char + 'WXYZ1234';
+
+    it('rejects passwords longer than 72 characters', function(done) {
+      try {
+        User.create({ email: 'b@c.com', password: pass73Char }, function(err) {
+          if (err) return done (err);
+          expect(err).to.have.property('message', /Password too long/);
+        });
+      } catch (e) {
+        done();
+      }
```

Comment:
@loay I walked through it a bit with you and said dig deeper, not give up and use try/catch as the solution. I find it hard to believe we must throw try/catch for a function that has a callback API. It is probably ok to move forward since you need this merged in ASAP and the previous behaviour also threw for the validation. Anyways, let's get @bajtos to chime in here.